azure_arc_data_jumpstart/eks/terraform/artifacts/DeployPostgreSQL.ps1

```diff
@@ -57,33 +57,33 @@ az deployment group create --resource-group $env:resourceGroup `
                            --no-wait
 Write-Host "`n"
 
-Do {
-    Write-Host "Waiting for PostgreSQL Hyperscale. Hold tight, this might take a few minutes..."
-    Start-Sleep -Seconds 45
-    $dcStatus = $(if(kubectl get postgresqls -n arc | Select-String "Ready" -Quiet){"Ready!"}Else{"Nope"})
-    } while ($dcStatus -eq "Nope")
-Write-Host "Azure Arc PostgreSQL Hyperscale is ready!"
-Write-Host "`n"
-
 # Ensures postgres container is initiated and ready to accept restores
+$pgControllerPodName = "jumpstartpsc0-0"
```

Comment:
I would change this to `$pgCoordinatorPodName` to stay aligned with [Citus](https://docs.citusdata.com/en/v7.0/aboutcitus/introduction_to_citus.html#architecture) terminology